packages/core/stories/patterns/menu-button/menu-button.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  StackLayout,
+  Text,
+  Display1,
+  Display2,
+  Display3,
+  Link,
+  Menu,
+  MenuItem,
+  MenuPanel,
+  MenuTrigger,
+  Button,
+  MenuGroup,
+  Input,
+  FormField,
+} from "@salt-ds/core";
+import {
+  ArrowUpIcon,
+  ArrowDownIcon,
+  MicroMenuIcon,
+  ChevronDownIcon,
+  UserIcon,
+  UserSolidIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+
+export default {
+  title: "Patterns/Menu Button",
+} as Meta;
+
+export const Regular = () => {
+  return (
+    <Menu>
+      <MenuTrigger>
+        <Button aria-label="Open Menu">
+          <StackLayout direction="row" gap={1}>
```

Comment:
Is there a reason to use a StackLayout here? I would expect Button's gap to already do this.

Reply:
it looked too close compared to uitk but maybe that's just how it is

Reply:
Spacing would have changed between UITK and Salt. These examples should match what is in the pattern but it hasn't been published yet?